### DISCUSSION TITLE
Can't buy or use copilot anymore

### DISCUSSION THREAD

**Discussion by Razikus** [community/community#45999] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Hello i cant purchase subscription for copilot
i have only this " Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended"
![Screenshot from 2023-02-01 10-09-28](https://user-images.githubusercontent.com/12664801/215999795-d3fe853d-ac93-4655-a618-399048d9e23b.png)

I have Github Pro

**Comment by jge162** [comment]
You need to reach out to support. [GitHub supprt](https://support.github.com/request/account?tags=hubberfy_account)

**Reply by jge162** [reply]
Account and billing option.
